### ISSUE TITLE
How to hide map from end user?

### ISSUE THREAD

**Issue by iamdubx** [BrockReece/vue-heatmapjs#1]
I followed your Docs but I don't see a way to hide heatmap from the user. It just always shows, even if I use 

```html
<div v-heatmap="show">
    <button @click="show = !show">Toggle Heatmap</button>
  </div>
```
It just overlays everything on the page, and buttons even doesn't work.

**Comment by iamdubx** [comment]
Thanks for response!

You can fix overlaying by adding `pointer-events: none` to `canvas` but I am not sure how this all is implemented. I guess I need to try vanilla version first.

But I was expecting not to see heatmap at all when I have `show: false` and `<div v-heatmap="show">`. For some reason it shows heatmap on initial load. Only after when I click button it hides it.

**Comment by BrockReece** [comment]
Thanks for the suggestion, that `pointer-events` rule sounds promising.
That is how I intended it to work, I will look into that as well.

**Comment by iamdubx** [comment]
Cool. Please tell if you need reproduction from my side. 
Can't wait to try out heatmaps in production already :smile:

**Comment by BrockReece** [comment]
So it looks like your hunch was correct. Adding this rule to my App.vue will allow you to click through the canvas.
```css
.heatmap-canvas {
  pointer-events: none;
}
```

I think I have figured out why it always initiates the heatmap too.
Cool, what is it that you are planning to use this for?

**Comment by iamdubx** [comment]
@BrockReece I am looking for few options: additional analytics build right in the Vue frontend server-rendered app (didn't tried it with Nuxt yet) and then want to try it in my backend admin panel. For second option I am still not sure how it can be implemented, tho.

**Comment by BrockReece** [comment]
Hi @iamdubx 

I have created a PR for these fixes, would you mind testing? It all looks good from my end though.

**Comment by iamdubx** [comment]
@BrockReece hey. I am not sure how to try out specific `canvas-overlay-issue` branch using npm?

**Comment by BrockReece** [comment]
I think this should do it.
`npm install https://github.com/BrockReece/vue-heatmapjs.git#canvas-overlay-issue`

Else give me a shout and I will create a prerelease tag through NPM

**Comment by iamdubx** [comment]
@BrockReece cool, I checked it out - all is working fine as I can see it!

**Comment by BrockReece** [comment]
@iamdubx thanks, this has been released in v1.0.2
